docassemble/assemblylinewizard/data/questions/pdf_field_tester.yml

```diff
@@ -133,9 +125,14 @@ subquestion: |
   
   ${my_pdf}
   
-  **Note: you do not need to download this form.** This is just a preview
-  to help you catch any errors. Use the original PDF that you uploaded
-  for the next step.
+  [FILE stop_sign, 5%] **Do not download this form.** It is just a preview
+  to help you catch any errors. 
```

Comment:
Now that the preflight and the weaver are combined, it's actually really clear that people won't upload this version mistakenly, because we don't ask them to upload new forms after this point.

IMO the solution is to add a very specific string to this preview file (i.e., DONT_UPLOAD_AGAIN), and check for that same string when people start the weaver from scratch. ("Hey, did you really mean to upload the document that's named DONT_UPLOAD_AGAIN?") The multiple stop signs seem really confusing, so if we can remove one that'd be great.

Reply:
The intention is to prevent "download" the pdf, not "upload" it again.

I'll leave the first stop sign for now so we can discuss on Monday as it's related to #134.

Reply:
I mean, the main reason for preventing people from downloading it is to prevent it from trying to use the weaver with it again. You're right though, we can wait til Monday to discuss more.